Analysis/Tasks/update_ccdb.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3
```

Comment:
Let's add a folder Analysis/scripts for this

Reply:
Ok

Reply:
Shall I add the CERN copyright statement?

Reply:
Yes

Reply:
Also please make sure you do not use `/usr/bin/python3` but `/usr/bin/env python3`.

Reply:
Thanks, it is now donw